### PR TITLE
Added Akto's Swag 

### PR DESCRIPTION
**Pull request by BandhiyaHardik**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

I have added Swag Opportunity of Akto.

Requirements:
1 merged PR: Akto sticker.
2 merged PRs: Akto t-shirt.
3 merged PRs: Akto hoodie.
4+ merged PRs: all of the above.

Issues: All the [Hacktoberfest tagged issues](https://github.com/akto-api-security/akto/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest)

This Fixes https://github.com/swapagarwal/swag-for-dev/issues/1091

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -17,6 +17,15 @@
     "dateAdded": "2019-10-04T15:29:00.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Akto",
+    "difficulty": "medium",
+    "description": "1 merged PR: Akto sticker,2 merged PRs: Akto t-shirt.,3 merged PRs: Akto hoodie.,4+ merged PRs: all of the above.All the Hacktoberfest tagged issues in Akto Repo in GitHub",
```
Comment (aslafy-z) [inline data.json:23]:
```suggestion
    "description": "Contribute to the Akto project in GitHub and earn cool merchandise! Simply make 1 merged PR to receive an Akto sticker, 2 merged PRs for an Akto t-shirt, 3 merged PRs for an Akto hoodie, and if you make 4 or more merged PRs, you'll get all of the above rewards.",
```

**Review by aslafy-z — CHANGES_REQUESTED** [review]

**Review by aslafy-z — APPROVED** [review]

File: data.json [hunk 2]
```diff
@@ -17,6 +17,15 @@
     "dateAdded": "2019-10-04T15:29:00.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Akto",
+    "difficulty": "medium",
+    "description": "Contribute to the Akto project in GitHub and earn cool merchandise! Simply make 1 merged PR to receive an Akto sticker, 2 merged PRs for an Akto t-shirt, 3 merged PRs for an Akto hoodie, and if you make 4 or more merged PRs, you'll get all of the above rewards.",
+    "reference": "https://hacktoberfestswaglist.com/list/#akto",
```
Comment (aslafy-z) [inline data.json:24]:
```suggestion
    "reference": "https://twitter.com/Aktodotio/status/1708716187582927316",
```

File: data.json [hunk 3]
```diff
@@ -17,6 +17,15 @@
     "dateAdded": "2019-10-04T15:29:00.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Akto",
+    "difficulty": "medium",
+    "description": "Contribute to the Akto project in GitHub and earn cool merchandise! Simply make 1 merged PR to receive an Akto sticker, 2 merged PRs for an Akto t-shirt, 3 merged PRs for an Akto hoodie, and if you make 4 or more merged PRs, you'll get all of the above rewards.",
+    "reference": "https://twitter.com/Aktodotio/status/1708716187582927316",
```
Comment (aslafy-z) [inline data.json:24]:
```suggestion
    "reference": "https://twitter.com/Aktodotio/status/1708716187582927316?t=EobNKKSUmfS3IX58a_jwZQ&s=19",
```

File: data.json [hunk 4]
```diff
@@ -17,6 +17,15 @@
     "dateAdded": "2019-10-04T15:29:00.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Akto",
+    "difficulty": "medium",
+    "description": "Contribute to the Akto project in GitHub and earn cool merchandise! Simply make 1 merged PR to receive an Akto sticker, 2 merged PRs for an Akto t-shirt, 3 merged PRs for an Akto hoodie, and if you make 4 or more merged PRs, you'll get all of the above rewards.",
+    "reference": "https://twitter.com/Aktodotio/status/1708716187582927316?t=EobNKKSUmfS3IX58a_jwZQ&s=19",
```
Comment (aslafy-z) [inline data.json:24]:
```suggestion
    "reference": "https://www.akto.io/akto-hacktoberfest-2023",
```

**Comment by BandhiyaHardik** [comment]
@aslafy-z  I have seen the checkings log i think there is some error related to mattermost regarding valid reference can you please look into it ?

image

**Comment by aslafy-z** [comment]
@BandhiyaHardik I seen it, I will open a PR to update the link. Your opportunity is live at devswag.io :)

**Comment by BandhiyaHardik** [comment]
Thank You @aslafy-z

**Pull request merged**